jujugui/static/gui/src/app/components/deployment-flow/expert-intro/expert-intro.js

```diff
@@ -27,32 +44,38 @@ class DeploymentExpertIntro extends React.Component {
   }
 
   render() {
-    return (
-      <DeploymentSection
-        instance="deployment-expert-intro-section">
+    const { entityModel } = this.props;
+    let content;
+    if (!entityModel) {
+      content = (
+        <div className="deployment-expert-intro__not-found">
+          This {this.isBundle ? 'bundle' : 'charm'} could not be found.
+          Visit the&nbsp;
+          <span className="link"
+            onClick={this._handleStoreClick.bind(this)}
+            role="button"
+            tabIndex="0">
+            store
+          </span>&nbsp;
+          to find more charms and bundles.
+        </div>);
+    } else {
+      const entity = entityModel.toEntity();
+      const machineNumber = this.isBundle ? entity.machineCount : 1;
+      content = (
         <div className="deployment-expert-intro">
           <div className="twelve-col">
             <div className="eight-col">
               <div className="deployment-expert-intro__section-title">
                 You are about to deploy:
               </div>
               <h2>
-                Saiku Drill
+                {entity.displayName}
               </h2>
               <div className="six-col">
-                <p>
-                  Get faster insights with less effort. Apache Drill is a
-                  schema-free SQL Query Engine.
-                </p>
-                <p>
-                  Saiku-Drill combines traditional SQL queries on modern,
-                  nontraditional datastores with the power of Saiku’s Business
-                  Intelligence engine.
-                </p>
-                <p>
-                  You can access multiple non-relational datastores directly,
-                  connecting to Hadoop, NoSQL or Cloud Storage.
-                </p>
+                <EntityContentDescription
```

Comment:
Can you also update this component to be a stateless component and then only pass in the description instead of the whole model?

Reply:
Done!